python/meta.py

```diff
@@ -0,0 +1,148 @@
+import mgp
+from mage.meta.parameters import Parameter
+from collections import defaultdict
+from typing import Dict, Tuple, Union, Iterator
+
+
+NodeKeyType = Tuple[str, ...]
+RelationshipKeyType = Tuple[NodeKeyType, str, NodeKeyType]
+
+
+class Counter:
+    def __init__(self, initial_value: int = 0):
+        self.total_count = initial_value
+        self.count_by_property_name = defaultdict(int)
+
+    def count(self) -> None:
+        self.total_count += 1
+
+    def count_property(self, property_name: str) -> None:
+        self.count_by_property_name[property_name] += 1
```

Comment:
I believe `increment`/`increment_property` are better names.

Reply:
Are you referring to the procedures' names or variables' names?

Reply:
The procedures

Reply:
Sure, makes sense and I agree. I will fix it now :D